.run_ci_tests.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# This script can be run manually (assuming docker-daemon access) as-is.
+# Or, export SPCCMD=bash to get your hands less clean.
+
+set -ex
+
+SPCCMD="${SPCCMD:-./.spc_ci_test.sh}"
+FQIN="docker.io/cevich/travis_ubuntu:latest"
+SPC_ARGS="--interactive --rm --privileged --ipc=host --pid=host --net=host"
+VOL_ARGS="-v ${REPO_VOL:-$PWD}:${REPO_VOL:-$PWD}:z
```

Comment:
No need for :z, since  you are running in a an SPC.

Reply:
Oh really?  Didn't know that, thanks!   Fixed.